lib/config/configItems.joi.js

```diff
@@ -150,6 +151,10 @@ const qpKafkaJoi = kafkaJoi.append({
     replication: extensionKafkaJoi,
 });
 
+const supportedLifecycleRulesJoi =  joi.array().items(
+    joi.string().valid(...supportedLifecycleRules))
+    .default(supportedLifecycleRules);
```

Comment:
Since this is specific to lifecycle, we might want to define it directly in `LifecycleConfigValidator` ?